api.py

```diff
@@ -332,8 +327,13 @@ def post(self, user_id):
                     INSERT INTO users_convos (convo_id, user_id)
                         VALUES (%s, %s)
                     '''
-            recipients_list = recipients.split(',')
-            for recipient_id in recipients_list:
+            for recipient in recipients_list:
```

Comment:
Why do we change from recipient_id, to username. You should be able to access those recipient_id's from your local cache bc you can only start conversations with contacts, therefore if you have a contact username, you have their user_id